ibis/bigquery/client.py

```diff
@@ -46,14 +55,30 @@ def __exit__(self, exc_type, exc_value, traceback):
 
 class BigQuery(Query):
 
+    def __init__(self, *args, **kwargs):
```

Comment:
How about explicit args?

```python
def __init__(self, client, ddl, params=None):
    super(BigQuery, self).__init__(client, ddl)
    self.query_parameters = params or []
```

Reply:
Good idea, will change. This originally was just so that I could do the default initialization and be robust to signature changes in the parent.